clayui.com/src/pages/404.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayIcon from '@clayui/icon';
+import {Link} from 'gatsby';
+import React from 'react';
+
+import clayLogo from '../../static/images/home/clay_logo.svg';
+import LayoutNav from '../components/LayoutNav';
+
+const spritemap = '/images/icons/icons.svg';
+
+const PageNotFound = () => {
+	return (
+		<div className="clay-404-page-container d-flex flex-column min-vh-100">
+			<header className="d-flex">
+				<div className="container-fluid-max-xl d-flex mb-0 ml-auto mr-auto mt-0 w-100">
+					<Link className="d-none d-sm-flex mx-3" to="/">
+						<img
+							alt="Clay"
+							className="align-items-center d-flex justify-content-center"
+							src={clayLogo}
+						/>
+					</Link>
+
+					<LayoutNav />
+				</div>
+			</header>
+
+			<div className="align-items-center container-fluid-max-lg d-flex justify-content-center m-auto main-content row">
+				<h1 className="col-12 display-3">
+					{"These aren't the components you're looking for."}
```

Comment:
Maybe we can reword this? I was looking for the page `ppp.html` but it doesn't exist.